native/packages/react-native-bpk-component-badge/src/BpkBadge.js

```diff
@@ -38,6 +38,9 @@ import { View, ViewPropTypes, StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
   viewBase: {
+    display: 'flex',
```

Comment:
Isn't `display: 'flex'` the default? You may be able to remove this line and it'll still work - worth a go!